src/ric/ric.h

```diff
@@ -91,6 +85,160 @@ enum RicTimers {
+    E_APPLY_MARIA_POWER_ANIM,        // RicEntityApplyMariaPowerAnim
+    E_80160C38,                      // func_80160C38
+    E_BLADE_DASH,                    // RicEntityBladeDash
+    E_801623E0,                      // func_801623E0
+    E_80162604,                      // func_80162604
+    E_MARIA,                         // RicEntityMaria
+    E_MARIA_POWERS,                  // RicEntityMariaPowers
+    E_80160F0C,                      // func_80160F0C
+    E_NOT_IMPLEMENTED_4,             // RicEntityNotImplemented4
+    E_BLINK_WHITE,                   // RicEntityPlayerBlinkWhite
+    E_SUBWPN_CRASH_CROSS_PARTICLES,  // RicEntitySubwpnCrashCrossParticles
+    E_801641A0,                      // func_801641A0
+    E_SHRINKING_POWERUP_RING,        // RicEntityShrinkingPowerUpRing
+    E_80167A70,                      // func_80167A70
+    E_SUBWPN_AXE,                    // RicEntitySubwpnAxe
+    E_CRASH_AXE,                     // RicEntityCrashAxe
+    E_SUBWPN_DAGGER,                 // RicEntitySubwpnDagger
+    E_80160D2C,                      // func_80160D2C
+    E_HIT_BY_ICE,                    // RicEntityHitByIce
+    E_HIT_BY_LIGHTNING,              // RicEntityHitByLightning
+    E_SUBWPN_REBOUND_STONE,          // RicEntitySubwpnReboundStone
+    E_SUBWPN_VIBHUTI,                // RicEntitySubwpnThrownVibhuti
+    E_SUBWPN_AGUNEA,                 // RicEntitySubwpnAgunea
+    E_SUBWPN_AGUNEA_HIT_ENEMY,       // RicEntityAguneaHitEnemy
+    E_CRASH_VIBHUTI,                 // RicEntityCrashVibhuti
+    E_CRASH_VIBHUTI_CLOUD,           // RicEntityVibhutiCrashCloud
+    E_CRASH_REBOUND_STONE,           // RicEntityCrashReboundStone
+    E_8016D9C4,                      // func_8016D9C4
+    E_CRASH_REBOUND_STONE_EXPLOSION, // RicEntityCrashReboundStoneExplosion
+    E_CRASH_BIBLE,                   // RicEntityCrashBible
+    E_CRASH_BIBLE_BEAM,              // RicEntityCrashBibleBeam
+    E_SUBWPN_BIBLE,                  // RicEntitySubpwnBible
+    E_SUBWPN_BIBLE_TRAIL,            // RicEntitySubpwnBibleTrail
+    E_SUBWPN_STOPWATCH,              // RicEntitySubwpnStopwatch
+    E_SUBWPN_STOPWATCH_CIRCLE,       // RicEntitySubwpnStopwatchCircle
+    E_801705EC,                      // func_801705EC
+    E_8016F198,                      // func_8016F198
+    E_AGUNEA_CIRCLE,                 // RicEntityAguneaCircle
+    E_AGUNEA_LIGHTNING,              // RicEntityAguneaLightning
+    E_CRASH_REBOUND_STONE_PARTICLES, // RicEntityCrashReboundStoneParticles
+    E_HIT_BY_DARK,                   // RicEntityHitByDark
+    E_HIT_BY_HOLY,                   // RicEntityHitByHoly
+    E_CRASH_STOPWATCH_DONE_PARTICLE, // RicEntityCrashStopwatchDoneSparkle
+    E_80170548,                      // func_80170548
+    E_TELEPORT,                      // RicEntityTeleport
+    E_DUMMY_66,                      // RicEntityDummy
+    NUM_ENTITIES,
+};
+
+enum RicBlueprints {
+    BP_BRAKE_SMOKE,
```

Comment:
Is this supposed to be Break? https://en.wikipedia.org/wiki/Brake

Reply:
I am not entirely sure how to say it in English and Brake was within the translation results I found: https://translate.google.it/?sl=it&tl=en&text=frenare&op=translate

What it is meant to represent is when the character is moving at high speed and it suddenly stops, there is the smoke coming from the ground.

Reply:
Brake works here then! Another option could be "skidding smoke".